diagnostic.js

```diff
@@ -29,29 +29,82 @@ mongoose.Promise = global.Promise;
 mongoose.connect('mongodb://localhost/mongoose-crud');
 
 /// ADD YOUR CODE BELOW
+const db = mongoose.connection;
+const Art = require('./art.js');
 
-const create = (name, description, startYear, endYear) => {};
+const done = function() {
+  db.close();
+};
+
+const create = (name, description, startYear, endYear) => {
 // Success -> Print new Movement as JSON
 // Failure -> Console.error
+Art.create({
+  name: name,
+  description: description,
+  startYear: startYear,
+  endYear: endYear,
+}).then(function(art) {
+  console.log(art);
+}).catch(function(error) {
+  console.error(error);
+}).then(done);
+};
 
-const index = () => {};
+const index = () => {
 // Success -> Print all Movements as JSON
 // Failure -> Console.error
+let search = {};
+if (arguments[0] && arguments[1]) {
+  let field = arguments[0];
+  let criterion = arguments[1];
+  if (criterion[0] === '/') {
+    let regex = new RegExp(criterion.slice(1, criterion.length - 1));
+    search[field] = regex;
+  } else {
+    search[field] = criterion;
+  }
+}
```

Comment:
It was not specified that we'd need to do any filtering in index, so all of this is unnecessary. However, it will not interfere with the desired behavior either.